qpc/scan/__init__.py

```diff
@@ -23,6 +23,7 @@
 SCAN_URI = "/api/v1/scans/"
 SCAN_BULK_DELETE_URI = "/api/v1/scans/bulk_delete/"
 SCAN_JOB_URI = "/api/v1/jobs/"
+SCAN_JOB_V2_URI = "/api/v2/jobs/"
```

Comment:
Unfortunately I have to use API v2 for scanjobs, as v1 does not support listing all scanjobs.

At the same time, I'm not sure if we want to port qpc to scanjobs v2 **now**.

Reply:
I think it is fine to migrate now

Reply:
As discussed on Slack, API v2 does not provide endpoint for canceling running scans.

In f21148c811f625ccdd30837da34012dfd5fe6026 I made v2 default for scanjobs, while leaving canceling on v1.